### DISCUSSION TITLE
Filter by Security Expressions

### DISCUSSION THREAD

**Discussion by Mark1161993** [querydsl/querydsl#2825] [Q&A]
Hi Guys,
I started using Querydsl following the Baeldung guide dedicated to Querydsl Web Support (https://www.baeldung.com/rest-api-search-querydsl-web-in-spring-data-jpa) and I'm enjoying it.

I have a question, is it possible to add a configuration to filter all queries on a entity using Spring-data Security Expressions?
I need to add a condition like: field = ?#{ principal?.id }.

Please help me, otherwise I cannot use this library
thank you!

**Comment by jwgmeligmeyling** [comment]
This question is probably better asked at Spring Data, as the integration between QueryDSL and Spring is maintained by them.
